app/src/main/res/layout/activity_main.xml

```diff
@@ -6,24 +6,43 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <FrameLayout
-        android:id="@+id/fragment"
+    <LinearLayout
```

Comment:
@NayakRoshan no nested viewgroups in `ConstraintLayout` unless absolutely necessary

Reply:
I used nested view group because to place views at the center of the screen. Is there any other way to get them to the center?